grpc/src/main/java/bisq/grpc/BisqGrpcClient.java

```diff
@@ -0,0 +1,298 @@
+    private final PlaceOfferGrpc.PlaceOfferBlockingStub placeOfferBlockingStub;
+    private final CorePersistenceProtoResolver corePersistenceProtoResolver;
+
+    public static void main(String[] args) throws Exception {
+        new BisqGrpcClient("localhost", 8888);
+    }
+
+    private BisqGrpcClient(String host, int port) {
+        this(ManagedChannelBuilder.forAddress(host, port)
+                // Channels are secure by default (via SSL/TLS). For the example we disable TLS to avoid
+                // needing certificates.
+                .usePlaintext(true).build());
+
+        // Simple input scanner
+        // TODO use some more sophisticated input processing with validation....
+        try (Scanner scanner = new Scanner(System.in);) {
+            while (true) {
+                long startTs = System.currentTimeMillis();
+
+                String[] tokens = scanner.nextLine().split(" ");
+                if (tokens.length == 0) {
+                    return;
+                }
+                String command = tokens[0];
+                List<String> params = new ArrayList<>();
+                if (tokens.length > 1) {
+                    params.addAll(Arrays.asList(tokens));
+                    params.remove(0);
+                }
+                String result = "";
+
+                switch (command) {
+                    case "getVersion":
+                        result = getVersion();
+                        break;
+                    case "getBalance":
+                        result = Coin.valueOf(getBalance()).toFriendlyString();
+                        break;
+                    case "getTradeStatistics":
+                        List<bisq.core.trade.statistics.TradeStatistics2> tradeStatistics = getTradeStatistics().stream()
+                                .map(bisq.core.trade.statistics.TradeStatistics2::fromProto)
+                                .collect(Collectors.toList());
+
+                        result = tradeStatistics.toString();
+                        break;
+                    case "getOffers":
+                        List<bisq.core.offer.Offer> offers = getOffers().stream()
+                                .map(bisq.core.offer.Offer::fromProto)
+                                .collect(Collectors.toList());
+                        result = offers.toString();
+                        break;
+                    case "getPaymentAccounts":
+                        List<PaymentAccount> paymentAccounts = getPaymentAccounts().stream()
+                                .map(proto -> PaymentAccount.fromProto(proto, corePersistenceProtoResolver))
+                                .collect(Collectors.toList());
+                        result = paymentAccounts.toString();
+                        break;
+                    case "placeOffer":
+                        // test input: placeOffer CNY BUY 750000000 true -0.2251 1000000 500000 0.12 5a972121-c30a-4b0e-b519-b17b63795d16
```

Comment:
NIT: 0.12 should be at least 0.15 otherwise the security deposit is too low for our current setup

Reply:
Fixed in d77ff101fbccda8ea3262fe41e53ded49d6b1bd6.